src/app/checkout/navigateToOrderConfirmation.tsx

```diff
@@ -2,10 +2,10 @@ import { noop } from 'lodash';
 
 import { isBuyNowCart } from '../common/utility';
 
-export default function navigateToOrderConfirmation(location = window.location, orderId?: number): Promise<never> {
+export default function navigateToOrderConfirmation(location = window.location, orderId?: number, isBuyNowCartEnabled?: boolean): Promise<never> {
```

Comment:
Here we can reorder params and not have `isBuyNowCartEnabled` as undefined.

Reply:
`location = window.location` seems for testing purpose only. We always use `window.location`. So I have removed `location` and kept the rest.